inc/sailag.h

```diff
@@ -141,6 +141,22 @@ typedef enum _sai_lag_attr_t
      */
     SAI_LAG_ATTR_DROP_TAGGED,
 
+    /**
+     * @brief LAG system port ID
+     *
+     * The application must manage the allocation of the system port IDs
+     * associated with the LAG for consistency across all switches in a
+     * VOQ based system. The system port ID range is from 1 to
+     * SAI_SWITCH_ATTR_NUMBER_OF_LAGS. The default value of 0 means this
+     * field is not used and SAI will allocate system port ID internally.
```

Comment:
Did you mean SAI will allocate the SYSTEM_PORT_AGGREGATE_ID internally.  "system port ID" means different thing.
Can we make it more explicit to say "LAG system port aggregate ID", instead of just saying system port ID in this comment?

Reply:
Sure.  Will make the comment explicitly say "system port aggregate ID".